ui/components/content-renderers/footnote-citation.js

```diff
@@ -44,14 +55,19 @@ export default class FootnoteCitation extends React.Component {
     const href = `#${this.props.content.footnote_node.identifier}`;
     const link = (
       <sup>
-        <Link className={klass} onClick={this.handleCitationClick} href={href}>
+        <a
```

Comment:
Is the problem with `Link` that setting its `ref` won't fire (since it's not an html tag)?

Reply:
So the weird thing is that according to the React docs (well, the React v16 docs) `ref` _is_ fired for components, it just passes the component instance rather than the DOM element. But what's strange is that for `Link`, the callback _is_ triggered but it's being passed neither a component instance nor a DOM element--it's just getting passed `null`.

That said, though, I do have an additional concern, which is that i'm uncertain whether a `Link` is even appropriate here.  My (admittedly novice) understanding of next.js's `Link` component is that it's meant for links that navigate to other pages on the site, rather than other `id`s on the current page; `Link` is required over `<a>` in such cases so that next can override the `onClick` handler to use `pushState` instead of the browser performing its own navigation.  However, in the case of links that navigate to different `id`s, the browser's default navigation _does_ work fine, so it seems to me like a `Link` shouldn't be needed.  But aside from that, there's also the fact that we're providing our _own_ `onClick()` behavior here, which renders `Link`'s `onClick` behavior particularly irrelevant. However, I could be mistaken given that I have very limited experience with next.js.

Reply:
Indeed, we don't need the extra functionality provided by Next's `Link`, but we've [tried](https://github.com/18F/omb-eregs/pull/547) to standardize on our wrapper over different types of anchors. If we can make this work with (our) `Link`, we should, if only to keep the code base consistent.

Reply:
D'oh! LOL, I just saw the name `Link` and assumed it was next.js's... I even looked at next.js's source to make sure `Link` was a full `React.Component` class and not a function, because `ref`s don't work with components that are functions!

Looking at our own `Link`, though, it appears that it _is_ a function, and that is most likely why `ref` isn't working.  Should I convert our `Link` into a standard `React.Component` subclass in this PR?

Reply:
Sounds good -- please add a comment about why when you do. I bet the linter will complain; adding such a comment when you disable the rule would be perfect.